includes/ajax.php

```diff
@@ -478,9 +478,19 @@ function tsml_ajax_geocode()
             if (!empty($meeting['end_time'])) add_post_meta($meeting_id, 'end_time', $meeting['end_time']);
         }
 
-        //add custom meeting fields if available
+         //add custom meeting fields if available
         foreach (['types', 'data_source', 'conference_url', 'conference_url_notes', 'conference_phone', 'conference_phone_notes'] as $key) {
-            if (!empty($meeting[$key])) add_post_meta($meeting_id, $key, $meeting[$key]);
+            if (!empty($meeting[$key])) {
+                add_post_meta($meeting_id, $key, $meeting[$key]);
+            }
+        }
+
+        //when found, update ACF custom meeting field ids
+        if (!empty($tsml_custom_meeting_fields)) {
+            $update_query = 'UPDATE ' . $wpdb->postmeta . ' ' .
+                'SET post_id = ' . $meeting_id . ' ' .
+                'WHERE post_id = ' . $meeting['id'];
+            $results = $wpdb->get_results($update_query);
```

Comment:
don't think we need these changes

Reply:
This change is the key to making it work. Not able to achieve the import of custom fields with it!